binaries/opae.io/opae/io/utils.py

```diff
@@ -51,7 +50,7 @@
 
 import libvfio
 
-PICKLE_FILE = '/var/lib/opae/opae.io.pickle'
+JSON_FILE = '/var/lib/opae/opae.io.json'
```

Comment:
we have add opae.io.json Spec file? required or not required ?

Reply:
Not required. We don't ship that file. It is created if it doesn't exist.